packages/react-native-app-auth/ios/RNAppAuth.m

```diff
@@ -714,6 +714,10 @@ - (NSString*)getErrorCode: (NSError*) error defaultCode: (NSString *) defaultCod
         @"firefox":
             ^{
                 return [OIDExternalUserAgentIOSCustomBrowser CustomBrowserFirefox];
+            },
+        @"edge":
```

Comment:
I get the following error when running `npx react-native run-ios` or when I try to build directly within Xcode: `No known class method for selector 'CustomBrowserEdge'`

Same here, I was able to confirm that I can open the URLs within the buttons in the Example app in Edge for iOS, but only after commenting this out.